src/XrdTpc/XrdTpcState.cc

```diff
@@ -90,9 +90,13 @@ bool State::InstallHandlers(CURL *curl) {
 }
 
 /**
- * Handle the 'Copy-Headers' feature
+ * Setup any headers necessary for the GET/PUT operation
+ *
+ * Currently includes:
+ * - Handle the 'Copy-Headers' feature
+ * - Adding `Expect: 100-continue` to get around a libcurl bug on uploads.
  */
-void State::CopyHeaders(XrdHttpExtReq &req) {
+void State::ConfigureHeaders(XrdHttpExtReq &req) {
```

Comment:
How about `SetupHeaders` instead of `ConfigureHeaders`? I think it makes more sense and in any case that's what you call it in your comment above.

Reply:
Done!